File: android/src/main/java/com/reactnativecloudpayments/EventEmitter.java
package com.reactnativecloudpayments;

import com.facebook.react.modules.core.DeviceEventManagerModule;
import com.facebook.react.bridge.WritableMap;
import androidx.annotation.Nullable;
import com.facebook.react.bridge.ReactContext;
import org.json.JSONObject;

public class EventEmitter {
  public EventEmitter() { }

  public void sendEvent(ReactContext reactContext, String eventName, @Nullable String params) {
    reactContext
      .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
      .emit(eventName, params);
  }
}

File: android/src/main/java/com/reactnativecloudpayments/EventEmitter.kt
package com.reactnativecloudpayments

import com.facebook.react.bridge.ReactContext
import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter

class EventEmitter {
  companion object {
    val listenerCryptogramCard = "listenerCryptogramCard"

    fun sendEvent(reactContext: ReactContext, eventName: String?, params: String?) {
      reactContext
        .getJSModule(RCTDeviceEventEmitter::class.java)
        .emit(eventName!!, params)
    }
  }
}
